migration/07900e6c41ab9a78917f66fcf7e0d9b8-8c84f81d9b09a5e0f028f3c6341ffb36/obelisk-upgrade

```diff
@@ -0,0 +1,15 @@
+# ob deploy: HTTPs support
```

Comment:
The contents of this file should be displayed when the user upgrades their project using `ob upgrade`.